spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -464,3 +464,22 @@ for (Pageable p = Pageable.ofSize(25).sortBy(Sort.desc("yearBorn"), Sort.asc("na
 }
 ----
 
+== Jakarta Data CDI Extension
+
+To run in environments with Jakarta Contexts and Dependency Injection (CDI), Jakarta Data providers each register their own CDI extension (`jakarta.enterprise.inject.spi.Extension`) to produce the bean instances that are defined via the `Repository` annotation and injected via the `Inject` annotation. The Jakarta Data specification employs several strategies for reducing and avoiding conflicts between Jakarta Data providers. The entity annotation class is used as the primary strategy. The Jakarta Data provider name serves as a secondary strategy.
+
+=== Entity Annotation Class
+
+The `jakarta.persistence.Entity` annotation from the Jakarta Persistence specification can be used by repository entity classes for Jakarta Data providers that are backed by a Jakarta Persistence provider. Jakarta Data providers that are not backed by a Jakarta Persistence provider must not support the `jakarta.persistence.Entity` annotation.
```

Comment:
Maybe it is an issue for a non-native speaker, but It has a cognitive load to understand this point:

> Jakarta Data providers not backed by a Jakarta Persistence provider must not support the `jakarta.persistence.Entity` annotation.